### ISSUE TITLE
Scroll bar and unlock icon covering menu items

### ISSUE THREAD

**Issue by OldSkoolVFX** [MrKepzie/Natron#784]
In using the last two v2.0 snapshots that actually opened in linux (Ubuntu 14.04) which I was supplied to work on *'Problem with Mask Bleed Through During Render #740'*.  The last commit id is f64efef1166bf709f6ddf69017b5bfcc202e41ea.

In these snapshots there is a **scrollbar** and an **unlock icon** covering the left side of the menu bar obscuring the options. The scrollbar and icon seem to be non-functional in that the don't appear to do anything and are both immovable so the menu options cannot be uncovered.  The menu is accessible by clicking on *'Layout'* and sliding the mouse cursor to the left to the hidden menus.

![natron2_odd_toolbar](https://cloud.githubusercontent.com/assets/10405914/8950489/50f3ea88-3590-11e5-8d1f-9c112db86343.png)

**Comment by MrKepzie** [comment]
I guess it's fixed in https://github.com/MrKepzie/Natron/commit/78eac4975ba0fc978257e7934e9b3fda2107d98d

**Comment by OldSkoolVFX** [comment]
This issue reappeared with the following snapshot:
*Natron-1d08991f7a2e24d086881c29b638461dd9db846e-Linux-x86_64bit*
Since it was considered closed I relooked at the issue and found that it was coming from my **_default.nl_** layout file.  I deleted it and rebuilt my layout and now the issue is not returning.  I think it is now resolved.  On hindsight, I think Natron was trying to cram the new Dopesheet into the older 1.2.1 layout.  **You might want to look at how Natron is working with older layouts when it gets upgraded.**
